### PR TITLE
added reproducibility info to docs

### PR DESCRIPTION
**Pull request by 00krishna**
- updated README
- updated make.jl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -541,3 +541,58 @@ for sparsity patterns. In the future, those two packages should just depend on
 ArrayInterface.jl and remove this issue entirely from the user space.
 
 Additionally, GPUs need ArrayInterfaceGPUArrays for proper determination of the indexing.
+
+## Reproducibility
+```@raw html
+<details><summary>The documentation of this SciML package was build using these direct dependencies,</summary>
```
Comment (ChrisRackauckas) [inline README.md:547]:
```suggestion
<details><summary>The documentation of this SciML package was built using these direct dependencies,</summary>
```

File: README.md [hunk 2]
```diff
@@ -541,3 +541,58 @@ for sparsity patterns. In the future, those two packages should just depend on
 ArrayInterface.jl and remove this issue entirely from the user space.
 
 Additionally, GPUs need ArrayInterfaceGPUArrays for proper determination of the indexing.
+
+## Reproducibility
+```@raw html
+<details><summary>The documentation of this SciML package was built using these direct dependencies,</summary>
```
Comment (ChrisRackauckas) [inline README.md:547]: wait, this is the README, not the docs.

Comment (00krishna) [reply README.md:547]: Ohh, yeah. Sorry. You are right. I need to put this into the index.md not the README. I just looked at the text and not the file name. Let me fix that then.

Comment (00krishna) [reply README.md]: Okay, this should be fixed now. I moved that text from the README to the index.md. Sorry again.

**Pull request merged**